modules/EmergencyMedicine/view/emergency-medicine/letter-generator/letter-generator.phtml

```diff
@@ -0,0 +1,9 @@
+<style>
```

Comment:
remove it

Reply:
same thing I kept it in order to have a template